src/pages/productlisting/product-listing.css

```diff
@@ -1,56 +1,83 @@
-.filter-label{
+.filter-label {
   width: max-content;
   cursor: pointer;
 }
 
 .product-page-body {
-    display: flex;
-  }
+  display: flex;
+}
 
 /* filter */
 
 .filters-container {
   width: 30rem;
   display: flex;
   flex-direction: column;
-  gap: 4rem;
+  gap: 2rem;
   padding: 3rem;
   flex-shrink: 10;
 }
 
-.filters-heading {
+.filter-close-btn{
+  min-width: unset;
+  min-height: unset;
+  width: 2.5rem;
+  height: 2.5rem;
+  display: none;
+  position: absolute;
+    right: 5px;
+    top: 5px;
```

Comment:
You can use shorthand property for this, it will reduced number of lines of code